x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/pages/index.tsx

```diff
@@ -99,19 +100,19 @@ export const MigrationRulesPage: React.FC<MigrationRulesPageProps> = React.memo(
       if (!migrationId || !migrationStats) {
         return <UnknownMigration />;
       }
-      if (migrationStats.status === SiemMigrationTaskStatus.FINISHED) {
-        return (
-          <MigrationRulesTable
-            migrationId={migrationId}
-            refetchData={refetchData}
-            integrations={integrations}
-            isIntegrationsLoading={isIntegrationsLoading}
-          />
-        );
-      }
       return (
         <RuleMigrationDataInputWrapper onFlyoutClosed={refetchData}>
           <>
+            <RuleMigrationsUploadMissingPanel migrationStats={migrationStats} topSpacerSize="s" />
```

Comment:
It would be better to show this one only when `status === FINISHED`. It's not possible to retry while `RUNNING`, and the `READY` panel already checks the missing resources internally

Reply:
done in https://github.com/elastic/kibana/pull/206658/commits/a2f80ceea14e6066d2fb73d0c90478534a7edd42